davicorreiajr/spotify-now-playing
# Master

## What does this PR do?

Adds the option to turn on notifications upon song changes.

When clicked on, notifications bring up Spotify.

## How to test?

Right click on the app's task bar icon and select the "Activate Notifications" option to have them appear when songs change on Spotify.

The "Activate Notifications" option is set off by default.
